README.md

```diff
@@ -48,6 +48,28 @@ valkey-search uses a **hybrid approach** with a query planner that selects the m
 - **Pre-filtering**
 - **Inline-filtering:** Filters results during the similarity search process.
 
+## Skip Index on RDB Load
+
+valkey-search includes an operational feature to mitigate issues in indexes. If set to true, index schema is loaded from rdb, but indexes are not loaded, they are created empty and will be re-filled by the backfill process.
+
+### Configuration
+
+The config is non-modifiable and should be provided to search module on load.
+```sh
+valkey-server --loadmodule /path/to/libsearch.so --rdb-load-skip-index yes
```

Comment:
Not necessarily a blocker for this change, but just thinking. The way that the current RDB format is setup, it looks like:

1. Minimum semantic version (e.g. 0.0.0 if any, 2.0.0 if you need 2.0 to load it)
2. Number of RDB sections
3. ... RDB sections ...

One thing I am thinking changes here is that the minimum semantic version relies on whether or not you care about the index contents or not. If we fundamentally change the index contents in version 2.0, but you don't care about it since you are loading in 1.0 with --rdb-load-skip-index, do we need to differentiate between the two?